docs/shared/migration/migration-angular.md

```diff
@@ -2,42 +2,47 @@
 
 Within an Nx workspace, you gain many capabilities that help you build applications and libraries using a monorepo approach. If you are currently using an Angular CLI workspace, you can transform it into an Nx workspace.
 
-{% youtube
-src="https://www.youtube.com/embed/dJG9zH30c-o"
-title="Nx in 100 seconds: Migrate an Angular CLI app to Nx"
-width="100%" /%}
-
-## Prerequisites
+## Migrating to a Standalone Angular App with Nx
 
-**The major version of your `Angular CLI` must align with the version of `Nx` you are upgrading to**. You can check the [Nx and Angular Version Compatibility Matrix](/angular-nx-version-matrix) for more information.
-
-## Using the Nx CLI while preserving the existing structure
-
-To use the Nx CLI in an existing Angular CLI workspace while keeping your existing file structure in place, use the `ng add` command with the `--preserve-angular-cli-layout` option:
+You can migrate to a [Standalone Angular App](/angular-standalone-tutorial/1-code-generation) with the command:
 
 ```shell
-ng add @nrwl/angular --preserve-angular-cli-layout
+nx init
 ```
 
-**Note**: If you specify a version of Nx (e.g. `ng add @nrwl/angular@13.10.0`), please make sure to use the appropriate command as shown in the compatibility table below:
+**Note**: To migrate to legacy versions of Nx, you'll need to refer to the following compatibility table below:
 
 | Nx version          | Collection to use | Flag to use                     | Example                                                       |
 | ------------------- | ----------------- | ------------------------------- | ------------------------------------------------------------- |
 | >= 13.10.0          | `@nrwl/angular`   | `--preserve-angular-cli-layout` | `ng add @nrwl/angular@13.10.0 --preserve-angular-cli-layout`  |
```

Comment:
Probably we want to add the max in the range for the Nx version here to clearly state up to which version they need to use this. It should be `>= 13.10.0 < 15.3.0`.